src/api/statements/JSONLDObject/index.js

```diff
@@ -40,34 +40,39 @@ const merge = (field, fields, currentOutput, data) => {
     return currentOutput;
 };
 
-export default function JSONLDObject(data, feed) {
+export default async function JSONLDObject(data, feed) {
     if (this.isLast()) {
         feed.close();
         return;
     }
-    const fields = this.getParam('fields', {});
+    const fields = this.getParam('fields', []);
     const collectionClass = this.getParam('collectionClass', '');
     const characteristics = this.getParam('characteristics', {});
     const exportDataset = this.getParam('exportDataset', false) === 'true';
-    const output = fields.filter(f => f.cover === 'collection').reduce(
-        (currentOutput, field) => {
+    const output = await fields.filter(f => f.cover === 'collection').reduce(
+        async (previousPromise, field) => {
```

Comment:
use composeAsync see next comment.
Just pass `() => Promise.resolve({ '@id': getUri(data.uri) });` as the first function